File: HotDesk_SPICE/TMS_HotDesk/App_Code/BaseUserControl.vb
Public Class BaseUserControl
    Inherits System.Web.UI.UserControl
#Region "Variables"
    Private _validationGroup As String

#End Region
#Region "Properties"
    Public Property ValidationGroup() As String
        Get
            Return Me._validationGroup
        End Get
        Set(ByVal value As String)
            Me._validationGroup = value
        End Set
    End Property
#End Region

End Class

File: HotDesk_PRAI/App_Code/BaseUserControl.cs
public class BaseUserControl : System.Web.UI.UserControl
{
    #region Variables
    private string _validationGroup;
    #endregion

    #region Properties
    public string ValidationGroup
    {
        get { return this._validationGroup; }
        set { this._validationGroup = value; }
    }
    #endregion
}
